README.md

```diff
@@ -48,7 +53,18 @@ on any transformations. All QGreenland GeoPackages and GeoTIFFs are projected
 in `EPSG:3413`.
 
 
-# Getting started
+# Level of Support
+
+This repository is fully supported by NSIDC. If you discover any problems or
+bugs, please submit an [Issue](https://github.com/nsidc/qgreenland/issues). If
+you would like to contribute to this repository, please see the section on
+[Contributing](#Contributing).
+
+See the [LICENSE](LICENSE) for details on permissions and warranties. Please
+contact nsidc@nsidc.org for more information.
+
+
+# Requirements
```

Comment:
"Requirements", "Installation", and "Usage" sections are all for QGreenland, _not_ the code here. Is that confusing? The layout of our ReadTheDocs does a good job of distinguishing between user and contributor documentation and includes all of the information here. We should really just have very top-level information here with links to RTD.

Reply:
>Is that confusing? 

Yes! I think we should keep the GitHub README focused on being the face of the code, and link out to RTD for user-targeted stuff.